segmentation_labeling_app/utils/query_utils.py

```diff
@@ -0,0 +1,113 @@
+import os
+from typing import Tuple
+from collections import namedtuple
+
+import pg8000
+import sqlite3
+from sqlite3 import Error
+
+
+class LabelingEnvException(Exception):
+    pass
+
+
+def get_labeling_env_vars():
+    """ returns a namedtuple with fields necessary for connection to
+    a postgres table. Fields are populated from environment variables
+    and non-credential fields have defaults.
+    """
+    try:
+        user = os.environ["LABELING_USER"]
+        password = os.environ["LABELING_PASSWORD"]
+    except KeyError:
+        raise LabelingEnvException(
+                "both env variables LABELING_USER and "
+                "LABELING_PASSWORD must be set")
+    host = os.environ.get('LABELING_HOST', "aibsdc-dev-db1")
+    database = os.environ.get(
+            'LABELING_DATABASE', "ophys_segmentation_labeling")
+    port = os.environ.get('LABELING_PORT', 5432)
+    LabelVars = namedtuple(
+            'LabelVars',
+            ['user', 'password', 'host', 'database', 'port'])
+    label_vars = LabelVars(user, password, host, database, port)
+    return label_vars
+
+
+def create_connection_sqlite(db_file):
+    """
+    Creates a database connection to the SQLite database
+    specified by db_file
+    Args:
+        db_file: database file
+
+    Returns:
+
+    """
+    conn = None
+    try:
+        conn = sqlite3.connect(db_file)
+    except Error as err:
```

Comment:
I'm not sure what you're attempting to catch here, but your try except is exactly the same as :
```conn = sqlite3.connect(db_file)```
It's going to except and print the exception? am i missing something?